R/print.check_model.R

```diff
@@ -141,8 +141,8 @@ print.see_check_model <- function(x,
 
   # make sure legend is properly sorted
   x$group <- factor(x$group, levels = c("low", "moderate", "high"))
-  levels(x$group) <- c("low (< 5)", "moderate (< 10)", "high (>= 10)")
-  names(colors) <- c("low (< 5)", "moderate (< 10)", "high (>= 10)")
+  levels(x$group) <- c("low (< 5)", "moderate (< 10)", "high (\u2265 10)")
```

Comment:
Will this also work on Windows?

Reply:
It does in my testing in RStudio view, R CMD graphics device, and knitting an Rmd document.